.travis.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         - sudo apt install -y texinfo libgif-dev libxpm-dev
         - travis_wait rustup install $(cat rust-toolchain)
       script:
-        - ./autogen.sh && ./configure
+        - ./autogen.sh && ./configure RUSTFLAGS="-D warnings"
```

Comment:
Traditionally, environment variables are specified before the command. So `RUSTFLAGS="-D warnings" ./configure`.

Reply:
It's not an environment variable, it's (meant to be) a variable used for the autoconf mechanisms, which is specified as a command line argument after `./configure`. But, anyway, it doesn't work because we force RUSTFLAGS to be empty in configure.ac.